gemgis/visualization.py

```diff
@@ -505,8 +505,15 @@ def create_mesh_from_cross_section(linestring: shapely.geometry.linestring.LineS
     # Creating PyVista PolyData
     surface = pv.PolyData(vertices, faces)
 
-    # Generating Tcoords
-    surface.t_coords = uv
+    # Set texture coordinates on PyVista mesh. This has been renamed a few times
+    # Reference https://github.com/pyvista/pyvista/issues/5209
+    try:
+        if pv._version.version_info < (0, 43):
+            surface.active_t_coords = uv
+        else:
+            surface.active_texture_coordinates = uv
```

Comment:
Maybe we want to add a third condition if the version is whenever `t_coords` was supported, but honestly that version of PyVista is not supported these days and this should use the modern API

Reply:
@banesullivan thanks for looking into that. I did not realize that the error was on my side during the mesh generation! I would actually keep it to the latest version of PyVista! If you don't mind just changing the old line 509 to what you have written in the new line 514 now (`surface.active_texture_coordinates = uv`), then I would be happy to merge it :)

Within the framework of the PyOpenSci Review, I will also have to extend and complete my tests.....

Reply:
Considering that 0.43 isn't out yet, I recommend keeping this

Reply:
Ah, right, should have read it more carefully. So it will be renamed in 0.43 again

Reply:
Definitely get this though and make sure this change is what you need. I did not run this code

Reply:
Just checked and your changes fix the issue! The texture is shown correctly again.